kythe/java/com/google/devtools/kythe/extractors/shared/ExtractorUtils.java

```diff
@@ -197,4 +199,40 @@ public static CompilationUnit normalizeCompilationUnit(CompilationUnit existingC
     existingCompilationUnit = builder.build();
     return existingCompilationUnit;
   }
+
+
+  public static String readEnvironmentVariable(String variableName) {
+    return readEnvironmentVariable(variableName, null);
+  }
```

Comment:
It looks like these are just a copy of EnvironmentUtils?

Reply:
Removed.